src/lib/block_if.c

```diff
@@ -618,17 +618,17 @@ blockif_open(const char *optstr, const char *ident)
 	sectsz = DEV_BSIZE;
 	psectsz = psectoff = 0;
 	candelete = geom = 0;
+  blocks = 0;
```

Comment:
Please indent to match the surrounding code (and again in the following if/ioctl combination).